isotracker.conf.in

```diff
@@ -0,0 +1,17 @@
+[general]
+url=https://iso.qa.ubuntu.com/xmlrpc.php
+username=@ISOTRACKER_USERNAME@
+password=@ISOTRACKER_PASSWORD@
+default_milestone=Jammy Daily
```

Comment:
This needs to be reworked. We avoided hardcoding anything (version, name…) in our repo, so that if a new release opened, we just receive an automated PR from our `detect-update-releases` workflow and we can review + merge.

This makes opening/tracking new releases painless compared to 60 places we need to update the name (and of course, forgot some in the wiki).
Can we retrieve the information from launchpad, or update maybe the release.csv file and add `+ " Daily"` to it?

Reply:
hm, you're right. Maybe we could actually make this super dynamic. A few ideas is: either generating the `isotracker.conf` file automatically in `notify-tracker` before creating the `ISOTracker` object or, maybe even better: tweaking `ISOTracker` in ubuntu-archive-tools to offer some functionality to pass the credentials and config data without the need of a file on the filesystem.

Anyway, good idea! I'll get that going.